asa/ja-jp/content/topics/adv_server_access/docs/gateway-configure.htm

```diff
@@ -593,8 +593,21 @@ <h3><a name="capture"></a>セッションキャプチャのオプション</h3>
+                                                                        <td class="TableStyle-standard-BodyD-Column1-Body2">
+                                                                            <p>最終的なセッションログの格納場所を指定します。ログは、ローカルに保存することも、AWSバケットまたはGCSバケットに保存することもできます。セッションログを複数の場所に格納する場合、ログは構成ファイルに指定されている順序で格納先に送信されます。セッションログは、<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>には絶対に送信されません。
+</p>
+                                                                            <p>S3ストレージを使用してセッションログを適切に記録・保存するには、S3バケットへのオブジェクトの書き込みに必要な権限をゲートウェイに構成します。それ以外の権限は必要ありません。                                </p>
+                                                                            <p>次のデフォルトの値と例を参照してください。</p>
+                                                                            <h5>例 - Linux/BSD</h5>
+                                                                            <p class="noteContent code">LogDestinations:<br />  - Type: file<br />    LogDir: /var/log/sft/sessions</p>
+                                                                            <div class="noteOkta">
+                                                                                <p class="noteContent">ゲートウェイは<code>/home</code>、<code>/root</code>、<code>/run/user</code>の各ディレクトリには書き込めません。</p>
+                                                                            </div>
+                                                                            <h5>例 - AWS</h5>
+                                                                            <p class="noteContent code">LogDestinations:<br />  - Type: s3<br />    Bucket: BUCKET-NAME<br />    Region: US-EAST-1<br />    # Use of ec2 instance IAM Role credentials for s3 bucket access is recommended<br />    # To use ec2 instance IAM Role credentials leave Profile, AccesKeyId, SecretKey, and SessionToken BLANK<br />    # Specify Profile for shared credentials<br />    Profile: AWS-PROFILE-NAME<br />    # Specify AccessKeyId, SecretKey, and SessionToken for static credentials<br />    AccessKeyId: SECRET<br />    SecretKey: SECRET<br />    SessionToken: SECRET</p>
```

Comment:
Making a note, will need to look at source. This is an odd way to format a code block.